modules/phase_field/src/materials/PFFracBulkRateMaterial.C

```diff
@@ -0,0 +1,31 @@
+#include "PFFracBulkRateMaterial.h"
+
+template<>
+InputParameters validParams<PFFracBulkRateMaterial>()
+{
+  InputParameters params = validParams<Material>();
+  params.addParam<FunctionName>("function", "", "Function describing energy release rate type parameter distribution");
+  params.addParam<Real>("gc", 1.0, "Energy release rate type parameter");
+
+  return params;
+}
+
+PFFracBulkRateMaterial::PFFracBulkRateMaterial(const std::string & name,
+                                               InputParameters parameters) :
+  Material(name, parameters),
+  _gc(getParam<Real>("gc")),
+  _gc_prop(declareProperty<Real>("gc_prop")),
+  _gc_prop_old(declarePropertyOld<Real>("gc_prop"))
```

Comment:
Do you get a warning when you use this Material? You declared an old property but didn't override initQpStateulProperties(). I realize you don't use the old property in your calculation so this isn't an error but if you want to remove the warning. Also I don't see where you are using this property anywhere. Is this leftover from earlier development?